packages/lib/src/paginator/Paginator.tsx

```diff
@@ -1,9 +1,66 @@
-import { useContext } from "react";
+import { useContext, useEffect, useRef, useState } from "react";
 import styled from "@emotion/styled";
 import DxcButton from "../button/Button";
 import DxcSelect from "../select/Select";
 import PaginatorPropsType from "./types";
 import { HalstackLanguageContext } from "../HalstackContext";
+import { responsiveSizes } from "../common/variables";
+import useWidth from "../utils/useWidth";
+
+const DxcPaginatorContainer = styled.div<{ width: number }>`
+  display: flex;
+  justify-content: ${({ width }) => (width && width <= Number(responsiveSizes.medium) * 16 ? "center" : "flex-end")};
```

Comment:
instead of checking the conditional everytime we could use a function or just a flag to avoid repeating the conditional so much.